lib/update.php

```diff
@@ -340,6 +361,28 @@ public function vimeography_addon_update_message($plugin_basename, $plugin_data)
   }
 
   /**
+   * Add a reminder to update an expiring key
+   * for the installed Vimeography theme / addon.
+   *
+   */
+  public function vimeography_key_expiring_message() {
+    $today = date("Y-m-d");
+
+
+    if($expiration_date < $today){
+      echo '<div class="notice notice-warning"><p>';
+        printf( __('Heads up! Your %1$s license is expiring soon! For continued support, activate your plugin <a href="http://www.vimeography.com/%2$s">here</a>', 'vimeography'), $this->expiring_addon->product_name, $this->expiring_addon->plugin_name );
```

Comment:
Can you change the message to read "To continue to receive updates and support, _click here to renew your license_"

We should also directly link the user to a cart with their license renewal added. The license renewal link format looks like this: https://vimeography.com/checkout/?edd_license_key={KEY}&download_id={ID}

These admin messages also need to be dismissable so that user can elect to hide them forever if they'd don't wanna see them again. We should have a Dismiss button that stores a flag in the database for a given message, and if that flag exists upon page load, do not show the message.